OptionSelector/CategoryTableViewController.m

```diff
@@ -0,0 +1,104 @@
+//
+//  CategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Artur Lan on 8/13/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryTableViewController.h"
+#import "CQCategory.h"
+#import "DetailTableViewController.h"
+
+@interface CategoryTableViewController ()
+
+@end
+
+@implementation CategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.category = @[@"Dogs", @"Cars", @"Food"];
```

Comment:
Don't you want to these to be instances of `CQCategory`, not `NSString`?

Also, when naming variables/properties that hold **collection data types** like arrays, it's usually preferable to use the plural. In this case, `categories` is a better name than `category`, which implies that it references a single category, not all of them.